### ISSUE TITLE
Provide macosx64 target

### ISSUE THREAD

**Issue by CommanderTvis** [MicroUtils/kotlin-logging#166]

**Comment by CommanderTvis** [comment]
As a workaround, one can publish the library to Maven Local:

```sh

#!/usr/bin/env sh

# Install kotlin-logging
mkdir -p kotlin-logging
cd kotlin-logging || exit
git init
git remote add origin https://github.com/MicroUtils/kotlin-logging.git
git fetch origin 7af97dbd4c79a3bf92cb4eacd4ca8abc0a6e217c
git reset --hard FETCH_HEAD
sed -i 's/\r$//' ./gradlew
chmod +x ./gradlew
./gradlew publishToMavenLocal -Dorg.gradle.jvmargs=-XX:MaxMetaspaceSize=512m --no-daemon --stacktrace
cd ..
```

**Comment by oshai** [comment]
Thanks for the workaround. I plan to add more targets after moving to circle ci and out of jcenter. Since this is a large effort it might take some time.

**Comment by CommanderTvis** [comment]
That's clear, publishing multiplatform libraries is cumbersome now. By the way, have a look at Github Actions workflows used by Stately (a pretty complex multiplatform library, which Native builds are published successfully) https://github.com/touchlab/Stately/blob/main/.github/workflows/deploy.yml

**Comment by HoffiMuc** [comment]
any news on this?

**Comment by CommanderTvis** [comment]
> any news on this?

Seems, no, because nothing has changed with publishing macOS targets. One still needs an Apple device.

**Comment by HoffiMuc** [comment]
concerning the workaround above...
I cloned master and had to comment out signing and detekt stuff in build.gradle.kts to get it compiled and publishToMavenLocal

BTW, I have a Mac (obviously) if I may help out somehow to also get the macosx64 target published, happy to lend a hand ;)

**Comment by oshai** [comment]
I also have mac. The main issue is that we want to make the release via tavis ci that does not support mac.

**Comment by CommanderTvis** [comment]
> I also have mac. The main issue is that we want to make the release via tavis ci that does not support mac.

Technically it [does](https://docs.travis-ci.com/user/reference/osx/).

**Comment by oshai** [comment]
oh, so maybe it wasn't like that when I checked before?
If it that simple I will add it. Of course you're also welcome to send PR (for the yaml config).

**Comment by oshai** [comment]
created PR #194 - so far it doesn't work, need to do some more tweaking probably.

**Comment by oshai** [comment]
fixed in `2.1.15`.

**Comment by CommanderTvis** [comment]
🚀 🚀 🚀 
Thank you. No more custom builds.
